tests/integration/api/health.rs

```diff
@@ -0,0 +1,19 @@
+//! Health endpoint integration tests
+
+use crate::integration::common::{client::RelayerClient, logging::init_test_logging};
+use tracing::info;
+
+/// Test that the health endpoint returns OK
+#[tokio::test]
+#[ignore = "Requires running relayer"]
+async fn test_health_endpoint() {
+    init_test_logging();
```

Comment:
init_test_logging is only needed if logs are used in tests? If not used this step is not needed?